Refine prometheus scrape in FAQ (#13803)
istio/istio.io: content/en/about/faq/metrics-and-logs/prometheus-application-metrics.md

weight: 90
---

> Yes. [Prometheus](https://prometheus.io/) is an open source monitoring system and time series database.
> You can start with [configuration for Prometheus](/docs/ops/integrations/prometheus/#Configuration) that enables collection of application.
> So that you can use Prometheus with Istio to record metrics that track the health of Istio and of
> applications within the service mesh. You can visualize metrics using tools like
> [Grafana](/docs/ops/integrations/grafana/) and [Kiali](/docs/tasks/observability/kiali/).

Reviewer: ```suggestion
Yes. [Prometheus](https://prometheus.io/) is an open source monitoring system and time series database.
You can use Prometheus with Istio to record metrics that track the health of Istio and of
applications within the service mesh. You can visualize metrics using tools like
[Grafana](/docs/ops/integrations/grafana/) and [Kiali](/docs/tasks/observability/kiali/). See [Configuration for Prometheus](/docs/ops/integrations/prometheus/#Configuration) to understand how to enable collection of metrics.
```
